ios/RNOneginiSdk.swift

```diff
@@ -261,7 +266,7 @@ class RNOneginiSdk: RCTEventEmitter, ConnectorToRNBridgeProtocol {
             if let error = error {
                 reject("\(error.code)", error.localizedDescription, error)
               } else {
-                resolve(true)
+                resolve(nil)
```

Comment:
if this `resolve` is not our code you can create an extension with a default param = nil, and than use it like resolve()